### PR TITLE
OLS-427 - Adding UI pipelines

### PR DESCRIPTION
**Pull request by JoaoFula**
Adding UI pipelines

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by tisnik — COMMENTED** [review]
ok

File: .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml [hunk 1]
```diff
@@ -0,0 +1,255 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/task: "[ols-installer, ols-e2e-task]"
+  name: ols-console-e2e-tests-pipeline
+spec:
+  description: |
+    This pipeline automates the process of running end-to-end tests for Openshift Lightspeed
```
Comment (kyoto) [inline .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml:10]:
```suggestion
    This pipeline automates the process of running end-to-end tests for OpenShift Lightspeed
```

File: .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    using a ROSA (Red Hat OpenShift Service on AWS) cluster. The pipeline provisions
+    the ROSA cluster, installs the openshift lightspeed operator using the installer, runs the tests, collects artifacts,
```
Comment (kyoto) [inline .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml:12]:
```suggestion
    the ROSA cluster, installs the OpenShift Lightspeed operator using the installer, runs the tests, collects artifacts,
```

File: .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml (hunk 1 above)
Comment (kyoto) [reply .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml:10]: Same in the other 2 files

File: .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml (hunk 2 above)
Comment (kyoto) [reply .tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml:12]: Same in the other 2 files

**Review by tisnik — COMMENTED** [review]
ok

**Comment by raptorsun** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / bundle"

**Comment by raptorsun** [comment]
/override "Red Hat Konflux / ols-e2e-operator-tests-416 / test-bundle"
/override "Red Hat Konflux / ols-e2e-service-tests-415 / test-bundle"
/override "Red Hat Konflux / ols-e2e-service-tests-416 / test-bundle"

**Pull request merged**